### PR TITLE
first cut at adapting MDI to use VRDR profiles

### PR DESCRIPTION
**Pull request by saulakravitz**
Needs more polishing, but good enough to demonstrate.

### CODE REVIEW

File: input/fsh/MDI/instances/Inline-Instance-for-bundle-mdi-to-edrs-a-freeman-11.fsh [hunk 1]
```diff
@@ -1,11 +1,6 @@
 Instance: Inline-Instance-for-bundle-mdi-to-edrs-a-freeman-11
 InstanceOf: USCorePractitionerProfile
```
Comment (ptran265) [inline input/fsh/MDI/instances/Inline-Instance-for-bundle-mdi-to-edrs-a-freeman-11.fsh:2]: should this be InstanceOf: PractitionerVitalRecordsNew

File: input/fsh/MDI/instances/Inline-Instance-for-bundle-mdi-to-edrs-a-freeman-12.fsh [hunk 2]
```diff
@@ -1,11 +1,6 @@
 Instance: Inline-Instance-for-bundle-mdi-to-edrs-a-freeman-12
 InstanceOf: USCorePractitionerProfile
```
Comment (ptran265) [inline input/fsh/MDI/instances/Inline-Instance-for-bundle-mdi-to-edrs-a-freeman-12.fsh:2]: should this be InstanceOf: PractitionerVitalRecordsNew

File: input/fsh/MDI/instances/us-core-patient-unknown-name.fsh [hunk 3]
```diff
@@ -18,4 +14,5 @@ Usage: #example
   * extension
```
Comment (ptran265) [inline input/fsh/MDI/instances/us-core-patient-unknown-name.fsh:14]: extension[DataAbsentReason], not using url

**Review by ptran265 — COMMENTED** [review]
I noticed the new files are not named with the -new naming convention. We might want to rename the files to make it more consistent and easier to navigate

File: input/fsh/MDI/instances/Inline-Instance-for-bundle-mdi-to-edrs-a-freeman-11.fsh (hunk 1 above)
Comment (saulakravitz) [reply input/fsh/MDI/instances/Inline-Instance-for-bundle-mdi-to-edrs-a-freeman-11.fsh]: Got rid of all of the inline instances....didn't rename example instances.

**Pull request merged**